### PR TITLE
Fix water tile in Six Flags Holland (and merge with Infernal Views fix)

### PR DESCRIPTION
**Pull request by HtotheTML**
Create similar fix for the missing water tile in Six Flags Holland.
hollandbefore
hollandafter

### CODE REVIEW

**Review by duncanspumpkin — APPROVED** [review]

File: distribution/changelog.txt [hunk 1]
```diff
@@ -65,7 +65,7 @@
 - Fix: [#19493] SV4 saves not importing the correct vehicle colours.
 - Fix: [#19517] Crash when peeps try to exit or enter hacked rides that have no waypoints specified.
 - Fix: [#19524] Staff counter shows incorrect values if there are more than 32767 staff members.
-- Fix: [#19641] Missing water tile in Infernal Views' river.
+- Fix: [#19641] Missing water tile in Infernal Views' and Six Flags Holland's river.
```
Comment (Broxzier) [inline distribution/changelog.txt:68]: This should also reference this PR now, #19643

File: src/openrct2/rct2/S6Importer.cpp [hunk 2]
```diff
@@ -642,6 +642,30 @@ namespace RCT2
...
+            }
+            if (String::Equals(_s6.ScenarioFilename, "Infernal Views.SC6", true)
+                || String::Equals(_s6.ScenarioFilename, "infernal views.sea", true))
+            {
+                auto surfaceElement = MapGetSurfaceElementAt(TileCoordsXY{ 45, 62 }.ToCoordsXY());
+
+                surfaceElement->SetWaterHeight(96);
+            }
+            else if (
+                String::Equals(_s6.ScenarioFilename, "Six Flags Holland.SC6")
```
Comment (Broxzier) [inline src/openrct2/rct2/S6Importer.cpp:659]: This one lacks the last argument. It is safe to assume the casing will match?

File: distribution/changelog.txt [hunk 3]
```diff
@@ -1683,4 +1683,4 @@ This is the first fully implemented version of OpenRCT2. RCT2.EXE is no longer r
 - Feature: RollerCoaster Tycoon 1 scenarios can now be opened in the scenario editor or by using the ‘edit’ command line action.
 - Feature: The ‘have fun’ objective can now be selected in the scenario editor.
 - Feature: Twitch integration.
-- Fix: Litter bins now get full and require emptying by handymen.
+- Fix: Litter bins now get full and require emptying by handymen.
```
Comment (Broxzier) [inline distribution/changelog.txt:1686]: Another unintended EOF change.

File: src/openrct2/rct2/S6Importer.cpp (hunk 2 above)
Comment (Broxzier) [reply src/openrct2/rct2/S6Importer.cpp:659]: Yes, that one. Did you leave it out deliberately?

Comment (HtotheTML) [reply src/openrct2/rct2/S6Importer.cpp:659]: i did not deliberately leave it out, no. it did build correctly without it, but again i can add it in if needed.

**Pull request merged**